src/ui/components/VPNRemoveDevicePopup.qml

```diff
@@ -130,17 +130,25 @@ Popup {
             VPNTextBlock {
                 id: popupText
 
+                property string textString
+
                 color: Theme.fontColor
                 font.pixelSize: 15
                 horizontalAlignment: Text.AlignHCenter
                 Layout.alignment: Qt.AlignHCenter
                 Layout.bottomMargin: Theme.vSpacing
+                Layout.leftMargin: Theme.popupMargin
+                Layout.rightMargin: Theme.popupMargin
                 Layout.fillWidth: true
-                lineHeight: 22
-
-                //: %1 is the name of the device being removed. The name is displayed on purpose on a new line.
-                //% "Please confirm you would like to remove\n%1."
-                text: qsTrId("vpn.devices.deviceRemovalConfirm").arg(popup.deviceName)
+                lineHeight: 22;
+                text: textString.arg(popup.deviceName)
+                wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+
+                Component.onCompleted: {
+                    //: %1 is the name of the device being removed. The name is displayed on purpose on a new line.
+                    //% "Please confirm you would like to remove\n%1."
+                    textString = qsTrId("vpn.devices.deviceRemovalConfirm").replace("\n", " ")
```

Comment:
I don‘t think this is optimal and the line break should be removed from the strings. Since we there is string freeze for `V2.5` this could be a temporary solution.